fbgen/src/PluginConfig.cmake

```diff
@@ -18,6 +18,8 @@ set(PLUGIN_CRX_KEY "MIGfMA0GCSqGSIb3DQEBAQUAA4GNADCBiQKBgQDOqMBmB8ThMqFT3/9s/bfS
 set(PLUGIN_CRX_ID "dlfdfkalmjbghacnadegnkmbcnpmnmfn")
 set(PLUGIN_CRX_WYRMHOLE_NAME ${COMPANY_NAME})
 set(PLUGIN_CRX_NATIVEHOST_NAME "${COMPANY_REVDOMAIN}.firewyrmhost")
+set(PLUGIN_CRX_NATIVEHOST_DESCRIPTION "${COMPANY_NAME} FireWyrm Native Messaging Wyrmhole")
+set(FBSTRING_HostFileName "FireWyrmNativeMessageHost")
 
```

Comment:
Wix installer needs a custom name set as FBSTRING_HostFileName but is only set in the nativemessagehost make script, if not here.